packages/es-components/src/components/patterns/screenReaderOnly/screenReaderOnly.md

```diff
@@ -2,7 +2,7 @@ This higher-order component allows components to be hidden from the user while b
 accessible to screen readers.
 
 ```
-const Hidden = screenReaderOnly('p');
+const Hidden = ScreenReaderOnly('p');
```

Comment:
I'm confused by this change. Did we change the import to require capitalization?